src/VisualStudio/Xaml/Impl/Implementation/LanguageClient/XamlCapabilities.cs

```diff
@@ -35,7 +35,7 @@ internal static class XamlCapabilities
                 OpenClose = false
             },
             SupportsDiagnosticRequests = true,
-            OnTypeRenameProvider = new DocumentOnTypeRenameOptions { WordPattern = OnTypeRenameHandler.NamePattern },
+            LinkedEditingRangeProvider = new LinkedEditingRangeOptions { /*WordPattern = OnTypeRenameHandler.NamePattern*/ },
```

Comment:
@mgoertz-msft please advise as to what should go here - there does not appear to be a word pattern parameter anymore with linked editing ranges capabilities.

Reply:
@dibarbet Removing it should be fine. Not much of a choice if it no longer exists there anyway. :-) I'll need to check the behavior later. Rest of XAML changes look good too. Thank you!

Reply:
```suggestion
            LinkedEditingRangeProvider = new LinkedEditingRangeOptions { },
```